pkg/controller/rpc/server.go

```diff
@@ -239,6 +244,27 @@ func (cs *ControllerServer) JournalList(ctx context.Context, req *ptypes.Journal
 	return &empty.Empty{}, nil
 }
 
+func (cs *ControllerServer) EngProfilerOp(ctx context.Context, req *ptypes.EngProfilerOpRequest) (*ptypes.EngProfilerOpReply, error) {
+	logrus.Infof("Profiler operation: %v, port (only used with enable): %v", req.Op, req.PortNumber)
```

Comment:
What does `port (only used with enable)` mean here? Isn't it straightforward? 

Suggest the log as below.
`Operating profiler: %v, port: %v`